chore: april community update blog

thulieblack opened the pull request:
Pr for April community update blog:
- AsyncAPI Bounty Program
- Code of Conduct
- Madrid Call for Speakers

thulieblack committed:
create folder

thulieblack committed:
Merge branch 'asyncapi:master' into master

thulieblack committed:
Delete aboutguide.md

thulieblack committed:
Merge branch 'asyncapi:master' into master

thulieblack committed:
banner+avatar

thulieblack committed:
update

thulieblack committed:
blog-metadata

thulieblack committed:
update

thulieblack committed:
update

thulieblack committed:
update

thulieblack committed:
update

thulieblack committed:
update

thulieblack committed:
update

thulieblack committed:
update

thulieblack committed:
update

thulieblack committed:
update

thulieblack committed:
update

thulieblack committed:
Merge branch 'master' of https://github.com/thulieblack/website

thulieblack committed:
draft1

thulieblack committed:
updates

thulieblack committed:
Merge branch 'asyncapi:master' into master

thulieblack committed:
update

thulieblack committed:
Merge branch 'master' of https://github.com/thulieblack/website

thulieblack commented:
This blog needs to be published by next week. Waiting on updates from @smoya and @jonaslagoni.

In the meantime did I miss anything @derberg? And @alequetzalli is the update okay with you?

derberg commented:
I think Ambassador Program article was released in April. So maybe promote that topic again and in article you can embed they youtube video you recorded with @Barbanio

thulieblack committed:
add-missing-sections

thulieblack committed:
Merge branch 'master' into master

thulieblack commented:
@alequetzalli some editorial review, please 😊

thulieblack committed:
add-spec-summary

thulieblack committed:
Merge branch 'master' into master

thulieblack committed:
Merge branch 'master' into master

thulieblack commented:
hey @alequetzalli ping pong

quetzalliwrites committed:
Merge branch 'master' into master

quetzalliwrites committed:
draft 2 

improved accessibility in anchor links, fixed grammar issues, and corrected people's names typos

quetzalliwrites reviewed (approved):
I edited the file myself and rewrote it to create Draft 2 for ya ✨ . I was trying to make it fast and easy for you to finally pub this 😄.

Also... wow! haha! I did NOT SEE THIS COMING AT ALL, you definitely didn't have to give my book free publicity here!! but.. I ACCEPT hahaha 🤣 Thank you for the kind words, they mean a lot. ♥️ 

_What I did:_ Improved accessibility for all anchor links, fixed grammar issues, and corrected misspelled Barbaño's name. (Remember you can always google her name and/or the Spanish `ñ` character when you need to write her name 😜 )

quetzalliwrites committed:
Merge branch 'master' into master

thulieblack commented:
@derberg can we publish now 😉

thulieblack committed:
Merge branch 'master' into master

akshatnema committed:
Merge branch 'master' into master

thulieblack committed:
Merge branch 'master' into master

thulieblack commented:
/rtm
